.travis.yml

```diff
@@ -9,7 +9,7 @@ cache: pip
 before_install:
     - pip install docutils
 script:
-    - make -j
+    - make -j$(nproc)
```

Comment:
AFAIR `nproc` reports 32 in the Travis CI boxes. I saw pytest/tox plugins defaulting to `2` instead if Travis env is detected because they only provide two cores.

Ref: https://docs.travis-ci.com/user/speeding-up-the-build/#makefile-optimization
Ref: https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system

cc: @ncoghlan

Reply:
In this particular case, "less than the hundreds of PEPs that may otherwise have their build processes launched in parallel" is enough to make it work reliably, but that's a good tip in general, thanks!

Reply:
I'm sharing this because in my experience if you use way more than 2 parallel processes in Travis it seriously slows down the build instead of speeding it up... At least this happened with pytest.

Reply:
> AFAIR `nproc` reports 32 in the Travis CI boxes.

All build VMs are 2-core.
https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system
https://travis-ci.org/native-api/test_travis/builds/564010159

Reply:
FTR I've done some testing and it turns out that Travis CI now reports the correct number of cores: https://travis-ci.com/webknjaz/peps/jobs/219946871#L703-L704

Reply:
@native-api there was a problem with this in the past, now it's good.